tuner/examples/test/tuner_test.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2024 Advanced Micro Devices, Inc
```

Comment:
This is the main relevant file in `examples/test`. The rest are just sample input IRs.